owlkettle/adw.nim

```diff
@@ -818,8 +849,20 @@ when AdwVersion >= (1, 2) or defined(owlkettleDocs):
     issueUrl: string
     website: string
     copyright: string
-    license: string
-    
+    license: string ## Sets the license as a custom text if it can’t be set via licenseType (when set licenseType will be set to GTK_LICENSE_CUSTOM).
+    licenseType: LicenseType ## Sets the license for from a list of known license.
+    legalSections: seq[LegalSection] ## Adds an extra section to the Legal page. Extra sections will be displayed below the application’s own information. The parameters copyright, licenseType and license will be used to present the it the same way as AboutWindow:copyright, AboutWindow:licenseType and AboutWindow:license are for the application’s own information. See those properties for more details. This can be useful to attribute the application dependencies or data.
+    applicationIcon: string
+    releaseNotes: string
+    comments: string
+    debugInfo: string
+    developers: seq[string]
+    designers: seq[string]
+    artists: seq[string]
+    documenters: seq[string]
+    credits: seq[(string, seq[string])] ## Adds additional credit sections with customizable titles
+    acknowledgements: seq[(string, seq[string])] ## Adds acknowledgment sections with customizable titles
+    links: seq[(string, string)] ## Adds additionals links placed in the details section
```

Comment:
I would use tuples with named fields (`tuple[x: float]`) instead of anonymous tuples. Otherwise the purpose of the fields is not immediately obvious.